The USES=libtool is adding dragonfly to "freebsd*)" in unwanted places.
This is an unintended side effect, avoid it by limiting to freebsd 10+.

--- configure.orig	2020-09-13 00:27:48 UTC
+++ configure
@@ -26405,7 +26405,7 @@ freebsd[45]*)
 $as_echo "#define C_BSD 1" >>confdefs.h
 
     ;;
-freebsd*)
+freebsd1*)
     if test "$have_pthreads" = "yes"; then
 	THREAD_LIBS="-lthr"
 	TH_SAFE="-thread-safe"
@@ -30081,7 +30081,7 @@ fi
 
 
 case "$host_os" in
-freebsd*)
+freebsd1*)
 { $as_echo "$as_me:${as_lineno-$LINENO}: checking for kinfo_getvmmap in -lutil" >&5
 $as_echo_n "checking for kinfo_getvmmap in -lutil... " >&6; }
 if ${ac_cv_lib_util_kinfo_getvmmap+:} false; then :
